### ISSUE TITLE
Uninitialized values are used for assignment operations

### ISSUE THREAD

**Issue by QiAnXinCodeSafe** [zerotier/ZeroTierOne#2419]
Hi all,
This is Qianxin CodeSafe Team, we found a suspicious issue,
Stored as an uninitialized value at 
https://github.com/zerotier/ZeroTierOne/blob/b5c51796b9a14ef0251d73e20e346683ad8014cf/attic/historic/anode/libanode/secure_random.c#L38
"0" is assigned to "i",and Entry into circulation,at
https://github.com/zerotier/ZeroTierOne/blob/b5c51796b9a14ef0251d73e20e346683ad8014cf/attic/historic/anode/libanode/secure_random.c#L38
The left expression of a compound assignment is an uninitialized value at
https://github.com/zerotier/ZeroTierOne/blob/b5c51796b9a14ef0251d73e20e346683ad8014cf/attic/historic/anode/libanode/secure_random.c#L57
To sum up,leading to s data problem

**Comment by glimberg** [comment]
Hi there,

Things under `/attic` are not actually part of ZeroTier One, nor are they used in it in any way.  IIRC, `anode` was a idea that was a predecessor to ZeroTier One.  It's kept around in the `/attic` folder for purely historic reasons.
